Formula/o/odin.rb

```diff
@@ -81,6 +81,32 @@ def install
           "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
         cp "libraygui.dylib", buildpath/raylib_installpath
       end
+    else
+      raylib_installpath = "vendor/raylib/linux"
+
+      # Create symbolic links for GLFW and Raylib libraries
+      ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/"vendor/glfw/lib/darwin/libglfw3.a"
+      ln_s Formula["raylib"].lib/"libraylib.a", buildpath/raylib_installpath/"libraylib.a"
+
+      # Adjust the name of the Raylib dynamic library for Linux
+      ln_s Formula["raylib"].lib/"libraylib.so.4.5.0", buildpath/raylib_installpath/"libraylib.so.500"
```

Comment:
You can also make this unconditional on the OS if you do:
```suggestion
      ln_s Formula["raylib"].lib/shared_library("libraylib", "4.5.0"), buildpath/raylib_installpath/shared_library("libraylib", "500")
```